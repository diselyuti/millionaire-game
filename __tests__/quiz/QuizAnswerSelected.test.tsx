import React from 'react';
import '@testing-library/jest-dom';
import QuizAnswerSelected from "@/app/components/quiz/QuizAnswerSelected.tsx";
import {render} from "@testing-library/react";

describe('QuizAnswerSelected', () => {
  test('should render texts', () => {
    const { container } = render(<QuizAnswerSelected letter="A" text="Answer" />);
    expect(container).toHaveTextContent('A');
    expect(container).toHaveTextContent('Answer');
  });

  test('should render QuizAnswerSelected', () => {
    const { container } = render(<QuizAnswerSelected letter="A" text="Answer" />);
    expect(container).toMatchSnapshot();
  });
});
