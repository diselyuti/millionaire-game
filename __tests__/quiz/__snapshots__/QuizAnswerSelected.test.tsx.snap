// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuizAnswerSelected should render QuizAnswerSelected 1`] = `
<div>
  <div
    class="hexagon selected"
  >
    <p
      class="answerText"
    >
      <span
        class="letter"
      >
        A
      </span>
      <span>
        Answer
      </span>
    </p>
  </div>
</div>
`;
